lib/routes/add_funds/deposit_to_btc_address_page.dart

```diff
@@ -195,8 +195,6 @@ class DepositToBTCAddressPageState extends State<DepositToBTCAddressPage> {
           .invoice_btc_address_warning_without_min_fee_account_connected(
         minSats,
         maxSats,
-        setUpFee,
```

Comment:
this has been reverted ref : https://github.com/breez/Breez-Translations/pull/17

Reply:
reverted